src/interactivity/win32/windowUiaProvider.cpp

```diff
@@ -12,56 +12,20 @@
 using namespace Microsoft::Console::Types;
 using namespace Microsoft::Console::Interactivity::Win32;
 
-WindowUiaProvider::WindowUiaProvider(IConsoleWindow* baseWindow) :
-    _pScreenInfoProvider{ nullptr },
-    WindowUiaProviderBase(baseWindow)
-{
-}
-
-WindowUiaProvider::~WindowUiaProvider()
-{
-    if (_pScreenInfoProvider)
-    {
-        _pScreenInfoProvider->Release();
-    }
-}
-
-WindowUiaProvider* WindowUiaProvider::Create(IConsoleWindow* baseWindow)
+HRESULT WindowUiaProvider::RuntimeClassInitialize(_In_ IConsoleWindow* baseWindow)
 {
-    WindowUiaProvider* pWindowProvider = nullptr;
-    ScreenInfoUiaProvider* pScreenInfoProvider = nullptr;
-    try
-    {
-        pWindowProvider = WRL::Make<WindowUiaProvider>(baseWindow).Detach();
+    RETURN_IF_FAILED(__super::RuntimeClassInitialize(baseWindow));

```

Comment:
Is `__super` really officially supported? I thought the "correct" way of doing this was to use the specific class you're trying to call the function from, like so:
```suggestion
    RETURN_IF_FAILED(WindowUiaProviderBase::RuntimeClassInitialize(baseWindow));
```

Reply:
I'll defer to you. It seems safer.